resources/views/resources-and-training.blade.php

```diff
@@ -60,13 +60,13 @@
         <div class="center center:wide stack stack:xl px-0">
             <h2 id="libraries">{{ __('Featured libraries') }}</h2>
             <x-interpretation name="{{ __('Featured libraries', [], 'en') }}" />
-            @if ($libraries->count() > 0)
+            @if ($featuredLibraries->count() > 0)
                 <div class="grid gap-6 md:grid-cols-2">
-                    @foreach ($libraries->take(4) as $library)
+                    @foreach ($featuredLibraries->take(4) as $library)
```

Comment:
Do we need the `take(4)` restriction here and for `featuredCollections`. They look like they are already being limited to 4. I'm worried that updates in the future may miss one of these changes; which could also have an impact on when to show the `Browse all` links.